### DISCUSSION TITLE
Possible to start Z2M without an adpater for testing purpose?

### DISCUSSION THREAD

**Discussion by Adesfire** [Koenkk/zigbee2mqtt#18767] [General]
Hello there!
I am currently setting up a new container on another Docker server, to be used as a backup if the master fails.
This way, I would just have to move my Dongle from the master to the secondary, restore the data folder and start the secondary container!

As I would be sure my new container is working properly regarding network and surrounding configuration, it would be nice for me to be able to start the container without having a proper Dongle connected to it.

So far, I tried to comment out the 'serial' part but Z2M, or to just start it without the Dongle connected, it won't start eitherways. Is there a way to do so?

`Zigbee2MQTT:error 2023-08-27 15:18:54: Error: Error while opening serialport 'Error: Error: No such file or directory, cannot open /dev/ttyACM0'
    at SerialPort.<anonymous> (/app/node_modules/zigbee-herdsman/src/adapter/z-stack/znp/znp.ts:146:28)
    at SerialPort._error (/app/node_modules/@serialport/stream/dist/index.js:75:22)
    at /app/node_modules/@serialport/stream/dist/index.js:111:18
`

Thank you very much :)

**Comment by Koenkk** [comment]
No this is not possible.

**Reply by Adesfire** [reply]
Thanks for the quick but disappointing reply!
